pages/company/index.vue

```diff
@@ -13,7 +12,10 @@
 <script setup>
 const config = useRuntimeConfig();
 
```

Comment:
By checking behaviors between useFetch() and $fetch() in my local via `npm run generate`,
I could not find any difference of those.
Would you check the difference as well?
If you don't find difference, we should keep original useFetch() one.